### PR TITLE
feat: release 2.3

### PR DESCRIPTION
**Pull request by spacewander**
Signed-off-by: spacewander <spacewanderlzx@gmail.com>

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +35,44 @@
...
+### Core
+
+- :sunrise: feat: support to communicate with etcd by TLS without verification in command line [#3415](https://github.com/apache/apisix/pull/3415)
+- :sunrise: feat: chaos test on route could still works when etcd is down [#3404](https://github.com/apache/apisix/pull/3404)
+- :sunrise: feat: ewma use p2c to improve performance [#3300](https://github.com/apache/apisix/pull/3300)
+- :sunrise: feat: support specifying https in upstream to talk with https backend [#3430](https://github.com/apache/apisix/pull/3430)
+- :sunrise: feat: allow customizing lua_package_path & lua_package_cpath [#3417](https://github.com/apache/apisix/pull/3417)
+- :sunrise: feat: allow to pass SNI in HTTPS proxy [#3420](https://github.com/apache/apisix/pull/3420)
+- :sunrise: feat: support gRPCS [#3411](https://github.com/apache/apisix/pull/3411)
+- :sunrise: feat: allow getting health check status via control API [#3345](https://github.com/apache/apisix/pull/3345)
```
Comment (membphis) [inline CHANGELOG.md:54]: "allow getting health check status via control API"
to
"allow getting upstream health check status via control API"

**Review by membphis — APPROVED** [review]
LGTM

**Review by moonming — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Comment by membphis** [comment]
ci failed

**Comment by juzhiyuan** [comment]
There have some conflicts, cc @spacewander

**Comment by membphis** [comment]
> There have some conflicts, cc @spacewander

we can fix this after the vote passed

**Pull request merged**